src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -574,6 +588,28 @@ JSC::EncodedJSValue SOCKET_BAD_PORT(JSC::ThrowScope& throwScope, JSC::JSGlobalOb
     return {};
 }
 
+JSC::EncodedJSValue UNCAUGHT_EXCEPTION_CAPTURE_ALREADY_SET(JSC::ThrowScope& throwScope, JSC::JSGlobalObject* globalObject)
+{
+    auto message = makeString("`process.setupUncaughtExceptionCapture()` was called while a capture callback was already active"_s);
+    throwScope.throwException(globalObject, createError(globalObject, ErrorCode::ERR_UNCAUGHT_EXCEPTION_CAPTURE_ALREADY_SET, message));
+    return {};
+}
+
+JSC::EncodedJSValue ASSERTION(JSC::ThrowScope& throwScope, JSC::JSGlobalObject* globalObject, JSC::JSValue msg)
```

Comment:
Instead of adding more functions, modify the one at the bottom to throw this message by default when `$ERR_ASSERTION` is passed through JS.

Reply:
Assuming this was meant for JS, otherwise ignore me if not